src/routes/blogs/[slug]/+page.svelte

```diff
@@ -2,16 +2,142 @@
   import type { PageData } from "./$types";
   import Marker from "$lib/components/Marker.svelte";
   import { rewriteHTML } from "$lib/utils/middlewares/rewrite-html";
+  import { formatDate } from "$lib/utils/services/datefmt";
 
   export let data: PageData;
 </script>
 
-<h1>
-  <Marker>blog</Marker>
-</h1>
+<div>
+  <div id="title">
+    <h1>
+      <Marker>blog</Marker>
+    </h1>
+    <div>
+      <h2>{data.detail.title}</h2>
+      <div>
+        {#each data.detail.tags as tag}
+          <span>{tag}</span>
+        {/each}
+      </div>
+    </div>
+  </div>
+  <div id="image">
+    <img src={data.detail.ogpImg.url} alt={data.detail.title} />
```

Comment:
```suggestion
    <img
      src={data.detail.ogpImg.url}
      alt={data.detail.title}
      height={data.detail.ogpImg.height}
      width={data.detail.ogpImg.width}
    />
```

https://github.com/misc-org/misc-mdn.dev/pull/10#discussion_r1581253163 と同様, width と height を渡してあげると嬉しいです.